docs/cli.md

```diff
@@ -14,7 +14,7 @@
 Well-formatted URL: 
 
 ```bash 
-./buildbench/tools/adaparse "http://www.google.com"
+adaparse "http://www.google.com"
```

Comment:
Should we name it just `ada` for the CLI or `ada-url` or `adaurl`?

Reply:
We can change the name.